addons/mail/static/tests/suggestion/suggestion.test.js

```diff
@@ -363,10 +358,10 @@ test("Internal user should be displayed first", async () => {
     await openFormView("res.partner", serverState.partnerId);
     await click("button", { text: "Send message" });
     await insertText(".o-mail-Composer-input", "@Person ");
-    await contains(":nth-child(1 of .o-mail-Composer-suggestion) strong", { text: "Person D" });
-    await contains(":nth-child(2 of .o-mail-Composer-suggestion) strong", { text: "Person C" });
-    await contains(":nth-child(3 of .o-mail-Composer-suggestion) strong", { text: "Person B" });
-    await contains(":nth-child(4 of .o-mail-Composer-suggestion) strong", { text: "Person A" });
+    await contains(":nth-child(1 of .o-mail-Composer-suggestion)", { text: "Person D" });
+    await contains(":nth-child(2 of .o-mail-Composer-suggestion)", { text: "Person C" });
+    await contains(":nth-child(3 of .o-mail-Composer-suggestion)", { text: "Person B" });
+    await contains(":nth-child(4 of .o-mail-Composer-suggestion)", { text: "Person A" });
```

Comment:
Same as previous comment: could take opportunity to `:eq()` syntax (short and more readable)